public/static/locales/en/form-builder.json

```diff
@@ -363,5 +363,12 @@
   "dyanamicRow": "Repeating question",
   "addToSet": "Add to set",
   "removeFromSet": "Remove from set",
-  "untitledForm": "Unnamed form file"
+  "untitledForm": "Unnamed form file",
+  "questionSet": {
+    "addAnother": {
+      "title": "Button",
+      "placeholder": "Add another",
```

Comment:
Could we possibly switch this to simply `Add` to avoid the gender-agreement issue?